sommaire/app/api/payments/route.ts

```
@@ -0,0 +1,48 @@
+export const POST = async (req: NextRequest) => {
+  const payload = await req.text();
+  const sig = req.headers.get("stripe-signature");
+  let event;
+  const endPointSecret = process.env.STRIPE_WEBHOOK_SECRET!;
+
+  try {
+    event = stripe.webhooks.constructEvent(payload, sig!, endPointSecret);
+    switch (event.type) {
+      case "checkout.session.completed":
+        console.log("Checkout session completed");
+        const sessionId = event.data.object.id;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)